lib/index.js

```diff
@@ -193,7 +193,17 @@ Nightwatch.cli = function(callback) {
       throw new Error('Supplied callback argument needs to be a function.');
     }
 
-    callback(argv);
+    if (argv.sequential){
+      // Enable running tests in sequential mode for supported and provided environments
+      let envs = argv.env && argv.env.split(',');
+      envs.reduce(
```

Comment:
Thanks @beatfactor I've noted it and will push the suggested changes.

Reply:
in any case, it doesn't look like this logic should be here. Maybe in `runner/cli/cli.js` or in `bin/runner.js`.

Reply:
Never mind, `reduce` needs to be used here, since it's a chain of promises. But you still should use the result I think. Try this (untested):

```
return envs.reduce((accumulator, env) => {
  return accumulator.then(_ => callback({...argv, env, e: env});
}), Promise.resolve());
``` 
And the change in [bin/runner.js](https://github.com/nightwatchjs/nightwatch/pull/3023/files#diff-601e5e20929b902752d6d093b229f4fc48630055e5397fb2099524fb6f879820) should not be necessary.